support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-id.html

```diff
@@ -0,0 +1,6 @@
+<p>
```

Comment:
There are a lot of anti-examples. [Here](https://github.com/jenkinsci/subversion-plugin/blob/master/src/main/resources/hudson/scm/SubversionSCM/AdditionalCredentials/help-realm.html) one of them.